tests/unit/providers/mariner/test_mariner.py

```diff
@@ -91,7 +91,7 @@ def disable_get_requests(monkeypatch):
     def disabled(*args, **kwargs):
         raise RuntimeError("requests disabled but HTTP GET attempted")
 
-    monkeypatch.setattr(parser.requests, "get", disabled)
+    monkeypatch.setattr(utils.http, "get", disabled)
```

Comment:
This could be something like this:
```python
@pytest.fixture()
def disable_get_requests(monkeypatch):
    def disabled(*args, **kwargs):
        raise RuntimeError("requests disabled but HTTP GET attempted")
    from vunnel import utils
    monkeypatch.setattr(utils.http, "get", disabled)
```

and put into `conftest.py` generically for anyone to use